addons/cookoff/functions/fnc_detonateAmmunition.sqf

```diff
@@ -31,7 +31,7 @@ private _magazine = _magazines select _magazineIndex;
 _magazine params ["_magazineClassname", "_amountOfMagazines"];
 
 if (_amountOfMagazines > 0) exitWith {
-    private _removed = _amountOfMagazines min floor(1 + random(missionNamespace getVariable [QGVAR(ammoScale), 6]));
+    private _removed = _amountOfMagazines min floor(1 + random(6 / GVAR(ammoCookoffDuration)));
```

Comment:
Only problem I have with this is that it's a divisor not a multiplier as suggested by the localised description string